Automattic/wp-calypso
# Odyssey Stats: Migrate create-odyssey-config.js to TypeScript - Migration Part 3

Related to #76570 

## Proposed Changes

Migrate `create-odyssey-config.js` to TypeScript. 

## Testing Instructions

* Ensure Odyssey Stats Dashboard and Widget works okay
* Ensure all tests pass `cd apps/odyssey-stats && yarn test:js`
* Ensure all GitHub checks pass

## Pre-merge Checklist
